glide.yaml

```diff
@@ -16,7 +16,7 @@ import:
 - package: github.com/m3db/prometheus_procfs
   version: 1878d9fbb537119d24b21ca07effd591627cd160
 - package: github.com/apache/thrift
-  version: 9549b25c77587b29be4e0b5c258221a4ed85d37a
+  version: ~0.10
```

Comment:
Can this match tchannel-go's dependency?
`>=0.9.3, <0.11.0`
https://github.com/uber/tchannel-go/blob/dev/glide.yaml#L11

Reply:
Done.